x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/job/process/autodetect/output/AutodetectResultProcessor.java

```diff
@@ -583,4 +585,8 @@ private boolean isAlive() {
     void setDeleteInterimRequired(boolean deleteInterimRequired) {
         this.deleteInterimRequired = deleteInterimRequired;
     }
+
+    public long currentRunBucketCount() {
```

Comment:
```suggestion
    long getCurrentRunBucketCount() {
```

I suggest making this package private because:

1. It's only used in a unit test
2. It's a bit of an esoteric thing to want to use elsewhere in the production code
3. The variable it returns is _not_ `volatile` on the basis that it's only used in one thread, but a caller of this method could violate that assumption